tests/fixtures/00-config-sample.toml

```diff
@@ -12,13 +12,7 @@ group = "workers"
 [cincinnati]
 base_url= "http://example.com:80/"
 
-[updates]
 # Valid strategies: immediate / periodic / remote_http
```

Comment:
Aside: what's the difference between `immediate` and `periodic` again?

Reply:
Heh OK right, the next unread notification answers this question: https://github.com/coreos/zincati/issues/34 :)